src/main/java/com/sap/oss/phosphor/fosstars/model/value/ArtifactVersionValue.java

```diff
@@ -0,0 +1,69 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import java.util.Objects;
+
+/**
+ * A value for a feature that provide a version for an artifact.
+ */
+public class ArtifactVersionValue extends AbstractValue<String> {
```

Comment:
I think this value class should hold `ArtifactVersion`.

```suggestion
public class ArtifactVersionValue extends AbstractValue<ArtifactVersion> {
```

Reply:
In this case the `ArtifactVersionValue` is only the version as a string and is used to provide the version of the artifact for which the security rating is calculated.
The `ArtifactVersion` is the version string together with the release date.

Reply:
With the change regarding `OssFeatures.ARTIFACT_VERSION` -> `OssFeatures.VERSION` the `ArtifactVersionValue` and `ArtifactVersionFeature` got removed.